src/components/APIDocs/APIDocs.js

```diff
@@ -1,10 +1,62 @@
+import {
+  Grid,
+  GridContainer,
+  CardGroup,
+  Card,
+  CardHeader,
+  CardBody,
+  CardFooter,
+  Button,
+} from "@trussworks/react-uswds";
 import React from "react";
+import { constants } from "../../helpers/constants";
+import { useNavigate } from "react-router-dom";
 
 export const APIDocs = () => {
+  const navigate = useNavigate();
+
   return (
-    <div>
-      <h1 style={{ textAlign: "left" }}>APIDocs</h1>
-      <p style={{ textAlign: "left" }}>Explore the CAM API Documentation</p>
-    </div>
+    <article>
+      <GridContainer>
+        <Grid row gap>
+          <Grid desktop={{ col: 8 }} className="text-left">
```

Comment:
Is this meant to be at the 8 or 12? The API Docs text (Explore the ...) currently covers 3/4 of the respective space when it could take up 100%. I think it would look better at 12, not sure what others think.